packages/web/src/scss/tools/_alignment.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:map';
+
+$_physical-to-logical-alignment-map: (
+    top: start,
```

Comment:
Hypothetically: In case extension this API with reverse flow, would it make sense to use `flex-start' or `flex-end' or not?

Reply:
As far as I know, `flex-*` values are replaced with plain `start`/`end`.

Reply:
OK, this is interesting:

> flex-start takes into account the presence of the -reverse values of the flex direction, while start does not.

https://stackoverflow.com/a/54278834

Didn't know that, thank you!

Anyway, we can solve it under the hood once (if) the support for reversed order is added.